errbot/backends/test.py

```diff
@@ -359,85 +345,10 @@ def tearDown(self):
 
     def assertCommand(self, command, response, timeout=5):
         """Assert the given command returns the given response"""
-        push_message(command)
-        self.assertIn(response, popMessage(), timeout)
+        self.bot.push_message(command)
+        self.assertIn(response, self.bot.pop_message(), timeout)
 
     def assertCommandFound(self, command, timeout=5):
         """Assert the given command does not exist"""
-        push_message(command)
-        self.assertNotIn('not found', popMessage(), timeout)
-
-
-@pytest.fixture
-def testbot(request):
```

Comment:
done, with the only nuance that the global push_message was a no-go.
People can use testbot.push_message now which I think is fine and kind of logical.